<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="11,0">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,0,0,3,2,2,5,2,32">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="labelSearch">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">min-width:280;
min-height:30;
color:black;</string>
        </property>
        <property name="text">
         <string>搜索信息</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,6,4">
        <item>
         <widget class="QLabel" name="labelSelectCity">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>选择城市：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSearchCity">
          <property name="placeholderText">
           <string>输入城市名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSearchCity">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,6,4">
        <item>
         <widget class="QLabel" name="labelSearchStation">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>搜索站点：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSearchStation">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>输入站点名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSearchStation">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelTransfer">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>换乘指南</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,12,16">
        <item>
         <widget class="QLabel" name="labelStart">
          <property name="text">
           <string>起点站(S)：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxStartLine">
          <item>
           <property name="text">
            <string>选择路线</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxStartStation">
          <item>
           <property name="text">
            <string>选择站点</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,12,16">
        <item>
         <widget class="QLabel" name="labelEnd">
          <property name="text">
           <string>终点站(E)：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxEndLine">
          <item>
           <property name="text">
            <string>选择路线</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxEndStation">
          <item>
           <property name="text">
            <string>选择站点</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>换乘策略</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButtonMinStation">
             <property name="text">
              <string>途径站数最少</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonMinTime">
             <property name="text">
              <string>搭乘时间最短</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonTransfer">
            <property name="text">
             <string>换乘</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonReset">
            <property name="text">
             <string>重置</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>换乘路线</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>工具(T)</string>
    </property>
    <widget class="QMenu" name="menu_add">
     <property name="title">
      <string>动态添加</string>
     </property>
     <addaction name="action_addall"/>
     <addaction name="action_addline"/>
     <addaction name="action_addstation"/>
     <addaction name="action_addconnect"/>
     <addaction name="separator"/>
     <addaction name="action_addbytext"/>
    </widget>
    <addaction name="action_transfer"/>
    <addaction name="menu_add"/>
    <addaction name="separator"/>
    <addaction name="action_changecity"/>
    <addaction name="action_queryinfo"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>视图(V)</string>
    </property>
    <addaction name="action_enlarge"/>
    <addaction name="action_shrink"/>
    <addaction name="action_wheel"/>
    <addaction name="separator"/>
    <addaction name="action_linemap"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>查看(W)</string>
    </property>
    <addaction name="action_toolbar"/>
    <addaction name="action_statusbar"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="action_usehelp"/>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>关于(A)</string>
    </property>
    <addaction name="action_qt"/>
    <addaction name="action_author"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_T"/>
   <addaction name="menu_V"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
   <addaction name="menu_A"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_enlarge"/>
   <addaction name="action_shrink"/>
   <addaction name="separator"/>
   <addaction name="action_addall"/>
   <addaction name="action_addline"/>
   <addaction name="action_addstation"/>
   <addaction name="action_addconnect"/>
   <addaction name="action_addbytext"/>
   <addaction name="separator"/>
   <addaction name="action_linemap"/>
  </widget>
  <action name="action_close">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_transfer">
   <property name="text">
    <string>换乘指南</string>
   </property>
  </action>
  <action name="action_queryinfo">
   <property name="text">
    <string>信息查询</string>
   </property>
  </action>
  <action name="action_addall">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/all.png</normaloff>:/icon/icon/all.png</iconset>
   </property>
   <property name="text">
    <string>所有(A)</string>
   </property>
  </action>
  <action name="action_addline">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/subway.png</normaloff>:/icon/icon/subway.png</iconset>
   </property>
   <property name="text">
    <string>线路(L)</string>
   </property>
  </action>
  <action name="action_addstation">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/station.png</normaloff>:/icon/icon/station.png</iconset>
   </property>
   <property name="text">
    <string>站点(C)</string>
   </property>
  </action>
  <action name="action_addconnect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/connect.png</normaloff>:/icon/icon/connect.png</iconset>
   </property>
   <property name="text">
    <string>连接(C)</string>
   </property>
  </action>
  <action name="action_addbytext">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/text.png</normaloff>:/icon/icon/text.png</iconset>
   </property>
   <property name="text">
    <string>文本方式(T)</string>
   </property>
  </action>
  <action name="action_enlarge">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/enlarge.png</normaloff>:/icon/icon/enlarge.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_shrink">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/shrink.png</normaloff>:/icon/icon/shrink.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_wheel">
   <property name="text">
    <string>鼠标可缩放</string>
   </property>
  </action>
  <action name="action_linemap">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/map.png</normaloff>:/icon/icon/map.png</iconset>
   </property>
   <property name="text">
    <string>地铁图</string>
   </property>
  </action>
  <action name="action_toolbar">
   <property name="text">
    <string>toolbar</string>
   </property>
  </action>
  <action name="action_statusbar">
   <property name="text">
    <string>statusbar</string>
   </property>
  </action>
  <action name="action_qt">
   <property name="text">
    <string>关于 Qt</string>
   </property>
  </action>
  <action name="action_author">
   <property name="text">
    <string>制作者</string>
   </property>
  </action>
  <action name="action_usehelp">
   <property name="text">
    <string>使用帮助</string>
   </property>
  </action>
  <action name="action_changecity">
   <property name="text">
    <string>更换城市</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
